src/core/service.c

```diff
@@ -1358,6 +1382,81 @@ static int service_collect_fds(
                 rfd_names[n_fds] = NULL;
         }
 
+        size_t n_openfile_fds = 0;
+        LIST_FOREACH(open_files, f, s->open_files) {
+                log_unit_debug(UNIT(s), "[service_collect_fds] openfile %s", f->path);
```

Comment:
We already record the function from which the message is being logged in a journald field, so we shouldn't repeat it in the log message itself.